internal/codegen/databasecodegen/row.go

```diff
@@ -171,6 +172,48 @@ func (g RowCodeGenerator) generateMutationForColumnsFunction(f *codegen.File) {
 	f.P("}")
 }
 
+func (g RowCodeGenerator) generateMutationForPresentColumnsFunction(f *codegen.File) {
+	f.P()
+	f.P("func (r *", g.Type(), ") MutatePresentColumns() (string, []string, []interface{}) {")
+	f.P("columns := make([]string, 0, len(r.", g.ColumnNamesMethod(), "()))")
+
```

Comment:
IMO I would skip blank lines in a small-ish function like this

Reply:
removed